<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertWindow</class>
 <widget class="QWidget" name="insertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加员工</string>
  </property>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>市场部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>研发部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>运维部</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>307</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>215</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>部门</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
